Makefile.am

```diff
@@ -6,16 +6,24 @@ sshfs_SOURCES = sshfs.c cache.c cache.h
 if FUSE_OPT_COMPAT
 sshfs_SOURCES += compat/fuse_opt.c compat/fuse_opt.h
 endif
+if DARWIN_COMPAT
+sshfs_SOURCES += compat/darwin_compat.c compat/darwin_compat.h
+endif
 
 sshfs_LDADD = $(SSHFS_LIBS)
 sshfs_CFLAGS = $(SSHFS_CFLAGS)
 sshfs_CPPFLAGS = -D_REENTRANT -DFUSE_USE_VERSION=26 -DLIBDIR=\"$(libdir)\"
 
-EXTRA_DIST = sshnodelay.c
-CLEANFILES = sshnodelay.so
+EXTRA_DIST = sshnodelay.c FAQ.txt
+CLEANFILES = sshnodelay.so sshfs.1 sshfs.1.tmp
 
 dist_man_MANS = sshfs.1
 
+sshfs.1: sshfs.1.in
+	$(AM_V_GEN)$(CPP) $(CPPFLAGS) -P -xassembler-with-cpp sshfs.1.in \
+	    | sed -e '/^$$/d' >sshfs.1.tmp || exit 1; \
+	mv sshfs.1.tmp sshfs.1
```

Comment:
Indeed it's an ugly hack. I didn't want to touch the code much as part of the merge... but considering this is brand new, I have redone this.

I've still kept the generation logic in the `Makefile` though. I personally think that using `AC_OUTPUT` to generate arbitrary files that are not part of the build system is wrong. Artifacts of the build should be generated by `Makefile` rules to avoid having to go through the possibly-expensive `configure` cycle just to regenerate such outputs. `Makefile`s are better at tracking finer-grained dependencies.